app/src/main/java/net/squanchy/schedule/domain/view/Schedule.kt

```diff
@@ -1,12 +1,37 @@
 package net.squanchy.schedule.domain.view
 
-data class Schedule(val pages: List<SchedulePage>) {
+import net.squanchy.support.system.CurrentTime
+import org.joda.time.DateTimeZone
+
+data class Schedule(val pages: List<SchedulePage>, val timezone: DateTimeZone) {
 
     val isEmpty: Boolean
         get() = pages.isEmpty()
 
     companion object {
 
-        fun create(pages: List<SchedulePage>) = Schedule(pages)
+        fun create(pages: List<SchedulePage>, timezone: DateTimeZone) = Schedule(pages, timezone)
     }
+
+    fun findTodayIndexOrDefault(currentTime: CurrentTime) =
+        pages
+            .indexOfFirst { page ->
+                val now = currentTime.currentDateTime().toDateTime(timezone)
+                page.date.toLocalDate().isEqual(now.toLocalDate())
+            }
+            .let {
+                when (it) {
+                    -1 -> 0 // default to the first page
```

Comment:
Nit: extract `-1` as `NOT_FOUND_INDEX` and `0` to `FIRST_PAGE_INDEX` so you can remove the comment